### ISSUE TITLE
Song titles cannot take input focus

### ISSUE THREAD

**Issue by IMP1** [glee-uk/virtualcampfire#6]
The song titles (and by extension the buttons for playing a song) cannot be focussed on, and so the user cannot tab between them. I'm unsure of the accessibility impacts of this but suspect there may be some.

I suggest having them be buttons (as they are more semantically buttons than links), which can be styled as not-buttons if desired. This should allow for tabbing between them. I also suggest changing the cursor when hovering over them to show that clicking will perform an action.

**Comment by timp** [comment]
This has changed, I think the change has fixed this issue.
